scripts/prepdocslib/listfilestrategy.py

```diff
@@ -23,12 +23,6 @@ def __init__(self, content: IO, acls: Optional[dict[str, list]] = None):
         self.content = content
         self.acls = acls or {}
 
-    def __enter__(self):
```

Comment:
We dont seem to be using this class as a context manager anywhere, so I deleted the code.

Reply:
this class looks like it should mostly be used as a context manager if it has a `.close()` method, so I'm surprised it's not being used in this way

Reply:
I think it'd be possible for it to be used that way, but it'd be a larger change affecting a few files. Right now it's used inside a try/finally, where close() is called in the finally clause.

cc @mattmsft